specs/policy.json

```diff
@@ -0,0 +1,85 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "Astarte Delivery Policy Schema",
+    "description": "This schema describes how an Astarte delivery policy should be declared.",
+    "type": "object",
+    "properties": {
+        "policy_name": {
+            "type": "string",
+            "pattern": "^([a-zA-Z][a-zA-Z0-9]*.([a-zA-Z0-9][a-zA-Z0-9-]*.)*)?[a-zA-Z][a-zA-Z0-9]*$",
+            "minLength": 1,
+            "maxLength": 128,
+            "description": "The name of the delivery policy. This has to be an unique, alphanumeric reverse internet domain name, shorther than 128 characters."
+        },
+        "handlers": {
+            "type": "array",
+            "items": {
+                "$ref": "#/$defs/handler"
+            },
+            "minItems": 0,
+            "maxItems": 200,
+            "uniqueItems": true,
+            "description": "An handler refers to one or more delivery errors and describes the retreat strategy Astarte should take when it/they occur."
+        },
+        "retry_times": {
+            "type": "integer",
+            "description": "The least amount of times Astarte will try to send an event (first excluded)."
+        },
+        "maximum_capacity": {
+            "type": "integer",
+            "description": "The maximum amount of events that can be stored in a queue."
+        },
+        "event_ttl": {
+            "type": "integer",
+            "description": "The lifetime of a message, in seconds."
+        }
+    },
+    "required": [
+        "policy_name",
+        "handlers",
+        "maximum_capacity"
+    ],
+    "$defs": {
+        "handler": {
+            "type": "object",
+            "properties": {
+                "on": {
+                    "anyOf": [
+                        {
+                            "type": "array",
+                            "items": {
+                                "type": "integer",
```

Comment:
You could add a range here since there's only a restricted range of valid values